spec/requests/markets_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "Markets", type: :request do
+  describe "GET /markets" do
+    it "works! (now write some real specs)" do
```

Comment:
Actually, I haven't. It's just that we haven't decided if we were going to use request specs. But Rails and Rspec core teams recommend to use request specs instead of controller spec. 
http://rspec.info/blog/2016/07/rspec-3-5-has-been-released/
"For new Rails apps: we don't recommend adding the rails-controller-testing gem to your application"

Knowing that, I'll write all tests from the controller specs as request specs.